src/js/homepage_items.js

```diff
@@ -41,4 +41,13 @@ const showItemsOnHomePage = () => {
   bindEventListenToUpdateButton(items);
 };
 
+const bindEventListenToResetButton = () => {
+  const resetButton = document.getElementById('reset-items-button');
+  resetButton.addEventListener('click', (e) => {
+    e.preventDefault();
+    showItemsOnHomePage();
+  });
+};
+
 showItemsOnHomePage();
+bindEventListenToResetButton();
```

Comment:
Works and looks good! You could try and wrap `showItemsOnHomePage()` and the `bindEventListen` functions into one function that loads the page instead calling them individually globally if you wanted, but not necessary, it's still easy to read.